onprc_ehr/src/org/labkey/onprc_ehr/notification/ColonyAlertsNotification.java

```diff
@@ -1591,27 +1590,29 @@ protected void nonContiguousHousing(final Container c, User u, final StringBuild
         QueryDefinition qd = us.getQueryDefForTable("HousingCheck");
         List<QueryException> errors = new ArrayList<>();
         TableInfo ti = qd.getTable(us, errors, true);
+        if (null == ti)
+            return;
 
         SQLFragment sql = ti.getFromSQL("hc");
         Map<String, Object> params = new HashMap<>();
         params.put("MINDATE", cal.getTime());
         QueryService.get().bindNamedParameters(sql, params);
-        sql = new SQLFragment("SELECT * FROM " + sql.getSQL(), sql.getParams());
+        sql = new SQLFragment("SELECT * FROM ").append(sql);
         QueryService.get().bindNamedParameters(sql, params);
```

Comment:
Probably harmless, but I don't think we need both calls to bindNamedParameters()

Reply:
Yup, the second one will definitely be a noop.  Either one will do the job, I'd probably delete the first one.